admin/admin-model.php

```diff
@@ -456,11 +338,13 @@ public function set_translation_in_mass( $type, $translations ) {
 	public function update_translations( $old_slug, $new_slug = '' ) {
 		global $wpdb;
 
-		$terms    = get_terms( array( 'taxonomy' => array( 'post_translations', 'term_translations' ) ) );
-		$term_ids = array();
-		$dr       = array();
-		$dt       = array();
-		$ut       = array();
+		$taxonomies = $this->translatable_objects->get_taxonomy_names( array( 'translations' ) );
+		$term_ids   = array();
+		$dr         = array();
+		$dt         = array();
+		$ut         = array();
+
+		$terms = get_terms( array( 'taxonomy' => $taxonomies ) );
```

Comment:
To slightly improve the readability:
```suggestion
		$term_ids = array();
		$dr       = array();
		$dt       = array();
		$ut       = array();

		$taxonomies = $this->translatable_objects->get_taxonomy_names( array( 'translations' ) );
		$terms      = get_terms( array( 'taxonomy' => $taxonomies ) );

```

Reply:
Done in b5d73a8151cf728cf599d09202e2b233bf0af954